lib/m2ts/caption-stream.js

```diff
@@ -451,6 +451,7 @@ var Cea608Stream = function(field, dataChannel) {
       this.mode_ = 'popOn';
 
     } else if (data === this.END_OF_CAPTION_) {
+      this.mode_ = 'popOn';
```

Comment:
It's been a while and I don't remember _exactly_ why, but it could be because [this](https://www.gpo.gov/fdsys/pkg/CFR-2007-title47-vol1/pdf/CFR-2007-title47-vol1-sec15-119.pdf) says:
> An End of Caption command leaves a paint-on caption fully intact in non-displayed memory. In other words, a paint-on style caption behaves precisely like a pop-on style caption which has been displayed.

If this is done, and in the absence of anything explicit, it's probably best to assume we'll continue with a pop-on caption.

Reply:
Oh, interesting. This seems like good info to put in a comment above this line since we're likely to forget later on